src/analyser.cpp

```diff
@@ -170,7 +170,12 @@ struct AnalyserInternalEquation
     static bool isKnownOdeVariable(const AnalyserInternalVariablePtr &odeVariable);
 
     static bool hasKnownVariables(const std::vector<AnalyserInternalVariablePtr> &variables);
+    bool hasKnownVariables();
+
+    static bool isNonConstantVariable(const AnalyserInternalVariablePtr &variable);
+
     static bool hasNonConstantVariables(const std::vector<AnalyserInternalVariablePtr> &variables);
+    bool hasNonConstantVariables();
```

Comment:
Is there a method to the spacing on these declarations?

Reply:
The first one is on its own because it's regarding known variables while the others are about non-constant variables. As for the second one, it's separated from the last two because it is an `isXxxx()` method while the others are `hasXxx()` methods.